### PR TITLE
kubeadm reference two file

### PR DESCRIPTION
**Pull request by YouthLab**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+
+<!-- Run this command in order to set up the Kubernetes master. -->
+运行这个命令来搭建Kubernetes master节点
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md]: 这个 => 此

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+<!-- ### Synopsis -->
+### 简介
+
+
+<!-- Run this command in order to set up the Kubernetes master. -->
+运行这个命令来搭建Kubernetes主节点。
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md]: 这个 => 此
中英文要有空格分隔

**Comment by YouthLab** [comment]
DONE the Fix, Please check again.

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
+
+<!-- Run this command in order to set up the Kubernetes master. -->
+运行此命令来搭建 Kubernetes master 节点
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md]: 这个看上去跟下面是一样的？

**Comment by zhangxiaoyu-zidif** [comment]
@YouthLab  could you squash by git cmd ?

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md (hunk 3 above)
Comment (tengqm) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md]: 英文版本是用程序生成的。

Comment (YouthLab) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md]: 暂时选择与英文版保持一致(*￣︶￣)

**Comment by YouthLab** [comment]
> @YouthLab could you squash by git cmd?
All right, I will do a squash. Thank you for your remind.

**Pull request merged**